<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditNoteDialog</class>
 <widget class="QDialog" name="EditNoteDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>1024</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add / Edit Note</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lblAddNoteBg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>560</width>
     <height>775</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>560</width>
     <height>775</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="m_ptr_instructions_label">
      <property name="text">
       <string>Add some notes:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="m_ptr_note_text_edit_box">
      <property name="minimumSize">
       <size>
        <width>440</width>
        <height>220</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>440</width>
        <height>225</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="plainText">
       <string>Enter textâ€¦</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_ptr_done_push_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>515</y>
     <width>104</width>
     <height>42</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblnoteTopBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>560</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>560</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_ptr_cancel_push_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>88</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>34</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblnoteSeperator">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>540</width>
     <height>1</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>540</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>540</width>
     <height>1</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="m_ptr_keyboard_background_gradient">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>560</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblwindowBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblwindowBg</zorder>
  <zorder>lblAddNoteBg</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>m_ptr_done_push_button</zorder>
  <zorder>lblnoteTopBar</zorder>
  <zorder>m_ptr_cancel_push_button</zorder>
  <zorder>lblnoteSeperator</zorder>
  <zorder>m_ptr_keyboard_background_gradient</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
